examples/COCO/my_threading.py

```diff
@@ -0,0 +1,91 @@
+from concurrent.futures import thread
```

Comment:
what is this file for? is it for data reading on Slurm?

Reply:
This is multi-threading package to speed up dataloading from storage. It is not dependent on slurm.